app/utils/prepare.js

```diff
@@ -13,12 +13,13 @@ type PrepareFn = (props: Object, dispatch: Dispatch<*>) => Promise<*>;
  */
 export default function prepare(
   prepareFn: PrepareFn,
-  watchProps: ?Array<string>
+  watchProps?: Array<string> = []
 ) {
   // Returns true if any of the given watchProps have changed:
   const componentWillReceiveProps = (oldProps, newProps) =>
-    watchProps &&
-    watchProps.some(key => get(oldProps, key) !== get(newProps, key));
+    watchProps
+      .concat('loggedIn')
+      .some(key => get(oldProps, key) !== get(newProps, key));
```

Comment:
Could this act weirdly if someone passes in `loggedIn` as a prop manually? You'd get a list like `['loggedIn', 'loggedIn']`. If that could cause issues I suppose we should remove any cases of `loggedIn` being passed as `watchProps` in this PR.

Reply:
The only thing that will happen is that the `loggedIn` prop will be checked twice.